packages/form_builder_phone_field/.fvm/fvm_config.json

```diff
@@ -0,0 +1,4 @@
+{
+  "flutterSdkVersion": "2.10.5",
+  "flavors": {}
+}
```

Comment:
Should we submit `fvm` environment subject? I am not sure about it.

Reply:
I don't know too :(
I feel it is difficult to know which versions support each packages or which ones they are built with. Maybe this isn't better solution for that